packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,97 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { SliderConfiguration, SliderOrientation } from "../slider";
+import {
+    Direction,
+    convertStylePropertyPixelsToNumber,
+} from "@microsoft/fast-web-utilities";
+import { convertPixelToPercent } from "../slider/slider-utilities";
+
+const defaultConfig: SliderConfiguration = {
+    min: 0,
+    max: 0,
+    direction: Direction.ltr,
+    orientation: SliderOrientation.horizontal,
+};
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    public root: HTMLDivElement;
+
+    @attr
+    public position: string;
+    private positionChanged(): void {
+        this.positionStyle = this.positionAsStyle();
+    }
+
+    @attr({ attribute: "hide-mark", mode: "boolean" })
+    public hideMark: boolean = false;
+    public config: SliderConfiguration = {
+        min: 0,
+        max: 0,
+        direction: Direction.ltr,
+        orientation: SliderOrientation.horizontal,
+    };
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.getSliderConfiguration();
+        this.setStyleForOrientation();
+        this.positionStyle = this.positionAsStyle();
+    }
+
+    private setStyleForOrientation = (): void => {
+        if (this.config.orientation === SliderOrientation.horizontal) {
+            this.classList.add("slider-label-horizontal");
+        } else {
+            this.classList.add("slider-label-vertical");
```

Comment:
do these classes need to be this verbose? Elsewhere we aren't prefixing with the control name. I'd prefer `horizontal` and `vertical` here (elsewhere regarding the prefix as well)